src/fight/computation_tools.rs

```diff
@@ -0,0 +1,517 @@
+use crate::fight::global_values;
+use crate::{model, regiment};
+
+/// Compute the value to hit the opponent
+///
+/// # Parameters
+/// offensive (usize): The offensive Stats of the attacking Regiment
+///
+/// defense (usize): The defense Stats of the defending Regiment
+pub fn compute_roll_to_hit(offensive: usize, defense: usize) -> usize {
+    let difference: i8 = offensive as i8 - defense as i8;
+
+    match difference {
+        i8::MIN..=-8 => 6,
+        -7..=-4 => 5,
+        -3..=0 => 4,
+        1..=3 => 3,
+        4..=i8::MAX => 2,
+    }
+}
+
+/// Compute the value to wound the opponent
+///
+/// # Parameters
+/// strength (usize): The strength Stats of the attacking Regiment
+///
+/// resilience (usize): The resilience Stats of the defending Regiment
+pub fn compute_roll_to_wound(strength: usize, resilience: usize) -> usize {
+    let difference: i8 = strength as i8 - resilience as i8;
+
+    match difference {
+        i8::MIN..=-2 => 6,
+        -1 => 5,
+        0 => 4,
+        1 => 3,
+        2..=i8::MAX => 2,
+    }
+}
+
+/// Compute the number of attacks of a Regiment
+///
+/// # Parameters
+/// regiment (regiment::Regiment): The Regiment attacking
+///
+/// # Return
+/// usize: The number of attacks of the Regiment
+pub fn get_nb_attacks(regiment_attacking: &regiment::Regiment) -> usize {
+    (regiment_attacking.get_model().get_stats().get_attack() as f64
+        * regiment_attacking.get_cols() as f64
+        * 1.5)
+        .floor() as usize
+}
+
+/// ## Compute the probability for a model to wound an another
+///
+/// ### Parameters
+/// (&Unit) main -> The attacker
+///
+/// (&Unit) target -> The defender
+///
+/// ### Return
+/// isize -> The probability for main to wound target
+pub fn compute_wound_probability(attacking: &model::Model, defending: &model::Model) -> f64 {
```

Comment:
The thing is that: with just one '#' I find the tip box created really unpleasant because of the fat size of the titles.
Actually I did not really respect any norm about those comments in this PR but I can propose this one 
`/// Compute the value to hit the opponent
///
/// ## Parameters
/// offensive (usize): The offensive Stats of the attacking Regiment
///
/// defense (usize): The defense Stats of the defending Regiment`
You can try it on your IDE and if it suits the rest of the team I'll update all of my functions comments for it.
Never seen the problem with the old norm because you never told us you were creating a comment norm and we only saw it on you refacto PR

Reply:
If you prefer the second one that's fine for me.
Can you modify the rest of the comment for the functions then ? So doc is the same everywhere

Reply:
Yes I agree we never discussed about it, i never even thought about it before the refacto but I think that we should ask to everyone if it suits to them before taking one.
And yes ofc if we all agree on one, I'll change the documentation on all my functions as said in my previous comment :+1: